### PR TITLE
feat: support converter for torch.log10

### PR DESCRIPTION
**Pull request by bowang007**
# Description
 
support converter for torch.log10

## Type of change
- New feature (non-breaking change which adds functionality)

# Checklist:

- [x] My code follows the style guidelines of this project (You can use the linters)
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas and hacks
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests to verify my fix or my feature
- [ ] New and existing unit tests pass locally with my changes
- [ ] I have added the relevant labels to my PR in so that relevant reviewers are notified

### CODE REVIEW

File: py/torch_tensorrt/dynamo/conversion/aten_ops_converters.py [hunk 1]
```diff
@@ -2683,3 +2683,20 @@ def aten_ops_scalar_tensor(
     return impl.unary.scalar_tensor(
         ctx, target, SourceIR.ATEN, name, args[0], dtype=kwargs.get("dtype")
     )
+
+
+@dynamo_tensorrt_converter(torch.log10)
```
Comment (zewenli98) [inline py/torch_tensorrt/dynamo/conversion/aten_ops_converters.py:2688]: Suggest changing to `torch.ops.aten.log10.default`

File: tests/py/dynamo/conversion/test_log10.py [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+            ((10,), torch.float),
+            ((1, 20), torch.float),
+            ((2, 3, 4), torch.float),
+            ((2, 3, 4, 5), torch.float),
+        ]
+    )
+    def test_log10_float(self, input_shape, dtype):
+        class log(nn.Module):
+            def forward(self, input):
+                return torch.log10(input)
```
Comment (zewenli98) [inline tests/py/dynamo/conversion/test_log10.py:21]: It would be more consistent with other converters to use `return torch.ops.aten.log10.default(input)`

File: tests/py/dynamo/conversion/test_log10.py [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+        [
+            ((10,), torch.int, 0, 5),
+            ((1, 20), torch.int32, -10, 10),
+            ((2, 3, 4), torch.int, -5, 5),
+        ]
+    )
+    def test_log10_int(self, input_shape, dtype, low, high):
+        class log(nn.Module):
+            def forward(self, input):
+                return torch.log10(input)
```
Comment (zewenli98) [inline tests/py/dynamo/conversion/test_log10.py:39]: Suggest using `return torch.ops.aten.log10.default(input)`

**Review by zewenli98 — COMMENTED** [review]
Overall looks good to me, just need some minor changes.

File: py/torch_tensorrt/dynamo/conversion/aten_ops_converters.py [hunk 4]
```diff
@@ -2708,6 +2708,23 @@ def aten_ops_scalar_tensor(
...
+    kwargs: Dict[str, Argument],
+    name: str,
+) -> Union[TRTTensor, Sequence[TRTTensor]]:
+    return impl.unary.log2(
+        ctx,
+        target,
+        SourceIR.ATEN,
+        name,
+        args[0],
+    )
```
Comment (gs-olive) [inline py/torch_tensorrt/dynamo/conversion/aten_ops_converters.py:2725]: Should this be `log10` for this PR?

Comment (bowang007) [reply py/torch_tensorrt/dynamo/conversion/aten_ops_converters.py:2725]: yeah looks like I was referring to the wrong PR when I was rebase onto master branch just now. Thanks

**Review by gs-olive — APPROVED** [review]
Looks good to me
